internal/store/agents.go

```diff
@@ -97,3 +105,25 @@ func (s *Store) DeleteAgentAPIKey(projectID, keyID string) error {
 		return fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// Agents encapsulates the logic to manage different cloud providers
+func (s *Store) AgentGlobalVersions() (*opsmngr.SoftwareVersions, error) {
+	switch s.service {
+	case config.OpsManagerService:
+		result, _, err := s.client.(*opsmngr.Client).Agents.GlobalVersions(context.Background())
+		return result, err
+	default:
+		return nil, fmt.Errorf("unsupported service: %s", s.service)
+	}
+}
+
+// Agents encapsulates the logic to manage different cloud providers
+func (s *Store) AgentProjectVersions(projectID string) (*opsmngr.AgentVersions, error) {
+	switch s.service {
+	case config.OpsManagerService:
+		result, _, err := s.client.(*opsmngr.Client).Agents.ProjectVersions(context.Background(), projectID)
+		return result, err
+	default:
+		return nil, fmt.Errorf("unsupported service: %s", s.service)
+	}
+}
```

Comment:
I don't see these two endpoints for Cloud Manager https://docs.cloudmanager.mongodb.com/reference/api/agents/

Reply:
AgentGlobalVersions is OM only but `AgentProjectVersions` should work with CM, if it's not documented let's leave like that